tests/foreman/cli/test_puppetmodule.py

```diff
@@ -24,30 +24,33 @@
 from robottelo.config import settings
 from robottelo.constants.repos import FAKE_0_PUPPET_REPO
 from robottelo.constants.repos import FAKE_1_PUPPET_REPO
-from robottelo.test import CLITestCase
 
 
-class PuppetModuleTestCase(CLITestCase):
-    """Tests for PuppetModule via Hammer CLI"""
+@pytest.fixture(scope='module')
+def make_setup():
+    org = make_org()
+    product = make_product({'organization-id': org['id']})
+    repo = make_repository(
+        {
+            'organization-id': org['id'],
+            'product-id': product['id'],
+            'content-type': 'puppet',
+            'url': FAKE_0_PUPPET_REPO,
+        }
+    )
+    Repository.synchronize({'id': repo['id']})
+    return {'org': org, 'product': product, 'repo': repo}
 
-    @classmethod
-    @pytest.mark.skipif((not settings.repos_hosting_url), reason='Missing repos_hosting_url')
-    def setUpClass(cls):
-        super().setUpClass()
-        cls.org = make_org()
-        cls.product = make_product({'organization-id': cls.org['id']})
-        cls.repo = make_repository(
-            {
-                'organization-id': cls.org['id'],
-                'product-id': cls.product['id'],
-                'content-type': 'puppet',
-                'url': FAKE_0_PUPPET_REPO,
-            }
-        )
-        Repository.synchronize({'id': cls.repo['id']})
+
+class TestPuppetModule:
```

Comment:
Consider dropping the class from the module, as it doesn't serve any functional purpose.

Reply:
Class dropped. Also added pytestmark as the skipif applies to all tests in the module. Is the usage ok?